packages/core/deploy/018_deploy_oracle_child_tunnel.js

```diff
@@ -0,0 +1,28 @@
+// Grabbed from official Polygon docs
+// https://docs.matic.network/docs/develop/l1-l2-communication/state-transfer/#pre-requisite
+const ADDRESSES_FOR_NETWORK = {
```

Comment:
nit: I think this could be done in fewer lines without wrapping onto new lines like this.

Reply:
Good point, done